package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.17.0",
     "react": "^16.0.0",
```

Comment:
if you don't use axios - remove it

you shouldn't have ANY code you don't use. I will be trash 1-2 month later